WordPressKitTests/WordPressComOAuthTests.swift

```diff
@@ -36,13 +38,16 @@ class WordPressComOAuthTests: XCTestCase {
 
         let expect = self.expectation(description: "One callback should be invoked")
         let client = WordPressComOAuthClient(clientID: "Fake", secret: "Fake")
-        client.authenticateWithUsername("fakeUser", password: "fakePass", multifactorCode: nil, success: { (token) in
+        client.authenticateWithUsername("fakeUser", password: "fakePass", multifactorCode: nil, needsMultifactor: { _, _ in
+            expect.fulfill()
+            XCTFail("This call should be successful")
+        }, success: { (token) in
```

Comment:
@Ecarrion @ThomazFB  Having to update the tests this way, that is by adding a parameter to satisfy the compiler, is usually a sign of a breaking change having been introduced if the method that is being modified is `public`.

That was indeed the case with this PR, see [this failed CI build](https://buildkite.com/automattic/woocommerce-ios/builds/16542#018b2c09-3960-48f0-93fb-b430349c1dd2/759-2868).

https://github.com/wordpress-mobile/WordPressKit-iOS/pull/632 is an attempt to restore backwards compatibility. More details on the rationale in the PR description.